assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+            "exitPoints": {
+                "SAMBO": ["RIDER", "SAITI", "SAMBO"]
+            },
+            "draw": [
+                ["SS901", "CUBIC"],
+                ["BUBLE", "CUBIC"],
+                ["CUBIC", "RIDER", "SAITI", "SAMBO*"]
+            ]
+        },
+        "DERBY3": {
+            "icao": "DERBY3",
+            "name": "Derby Three",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "SS901", "ANEMO", "EBOSI"],
+                "RJSS27": [["_RWY0901DME", "A5+"], "SS701", ["_SS701233001", "A13+"], "EBOSI"]
+            },
+            "body": ["DERBY"],
+            "exitPoints": {
+                "GTC": ["GTC"]
+            },
+            "draw": [
+                ["SS901", "ANEMO", "EBOSI"],
+                ["SS701", "EBOSI"],
+                ["EBOSI", "DERBY", "GTC*"]
+            ]
+        },
+        "IXE8": {
+            "icao": "IXE8",
+            "name": "Iwaki Eight",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "^_RWY2703DME", "_SDE120005"],
+                "RJSS12": [],
+                "RJSS27": ["^_SDE120005"],
+                "RJSS30": ["_SDE120005"]
+            },
+            "body": ["_IXE024SDE120", ["_IXE024047", "A110+"], ["_IXE024028", "A150+"]],
+            "exitPoints": {
+                "IXE": ["IXE"]
+            },
+            "draw": [
+                ["_SDE120005", "_IXE024SDE120", "IXE"]
+            ]
+        },
+        "SDE6R": {
+            "icao": "SDE6R",
+            "name": "Sendai Reversal Six",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "^_RWY2703DME", "_SDE120005", "^_SDE120010"],
+                "RJSS12": ["^_SDE120010"],
+                "RJSS27": [["_RWY0902DME", "A5+"], "_SDE120005", "^_SDE120010"],
+                "RJSS30": ["_SDE120005", "^_SDE120010"]
+            },
+            "body": [],
+            "exitPoints": {
+                "SDE": [["SDE", "A70+"]]
+            },
+            "draw": [
+                ["_SDE120010", "SDE"]
+            ]
+        },
+        "STEED3": {
+            "icao": "STEED3",
+            "name": "Steed Three",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "SS901", "STEED"],
+                "RJSS27": [["_RWY0902DME", "A5+"], "BUBLE", "STEED"]
+            },
+            "body": [],
+            "exitPoints": {
+                "RIKYU": ["RIKYU"]
+            },
+            "draw": [
+                ["SS901", "STEED"],
+                ["BUBLE", "STEED"],
+                ["STEED", "RIKYU*"]
+            ]
+        }
+    },
+    "stars": {
+        "LANCEA": {
+            "icao": "LANCEA",
+            "name": "Lance East Alfa",
+            "entryPoints": {
+                "LANCE": [["LANCE", "A110+"]]
+            },
+            "body": [["FUBOU", "A60+"], ["SNOOK", "A40+"], ["TOPAZ", "S230-"], ["PERID", "A15+"]],
+            "rwy": {
+                "RJSS27": ["#360"]
```

Comment:
No need for the heading here, aircraft will continue on their present heading anyway